alchemist/alchemist-incarnation-biochemistry/src/test/kotlin/it/unibo/alchemist/test/TestMoleculeSwapWithinNeighborhood.kt

```diff
@@ -98,30 +96,29 @@ private fun startSimulation() {
     simulation.run()
 }
 
-private fun matcher(test: (Reaction<Double>) -> Result) = object: Matcher<Reaction<Double>> {
+private fun matcher(test: (Reaction<Double>) -> Result) = object : Matcher<Reaction<Double>> {
     override fun test(reaction: Reaction<Double>) = test.invoke(reaction)
 }
 
 private val Int.conditions: Matcher<Reaction<Double>>
-    get() = matcher{ Result(
+    get() = matcher { Result(
             it.conditions.size == this@conditions,
-            "reaction should have ${this@conditions} conditions but it has ${it.conditions.size}",
-            "reaction should not have ${this@conditions} conditions but it has"
-    )}
+            "reaction should have ${ this@conditions } conditions but it has ${ it.conditions.size }",
+            "reaction should not have ${ this@conditions } conditions but it has"
+    ) }
 
 private val Int.neighborConditions: Matcher<Reaction<Double>>
-    get() = matcher{ Result(
-            it.conditions.filter{ c -> c is AbstractNeighborCondition }.size == this@neighborConditions,
-            "reaction should have ${this@neighborConditions} neighbor conditions but it has " +
-                    "${it.conditions.filter{ c -> c is AbstractNeighborCondition }.size}",
-            "reaction should not have ${this@neighborConditions} conditions but it has"
-    )}
-
+    get() = matcher { Result(
+            it.conditions.filter { c -> c is AbstractNeighborCondition }.size == this@neighborConditions,
+            "reaction should have ${ this@neighborConditions } neighbor conditions but it has " +
+                    "${ it.conditions.filter { c -> c is AbstractNeighborCondition }.size }",
+            "reaction should not have ${ this@neighborConditions } conditions but it has"
+    ) }
```

Comment:
Maybe a class extending `Result` (or decorating it) here would save us the duplication of `Int.conditions` and `Int.neighborConditions`